lib/Github/Api/User/Migration.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Github\Api\User;
+
+use Github\Api\AbstractApi;
+use Github\Exception\MissingArgumentException;
+
+class Migration extends AbstractApi
+{
+    /**
+     * @link https://docs.github.com/en/rest/migrations/users?apiVersion=2022-11-28#list-user-migrations
+     *
+     * @param array $params
+     *
+     * @return array|string
+     */
+    public function list(array $params = [])
+    {
+        return $this->get('/user/migrations', $params);
+    }
+
+    /**
+     * @link https://docs.github.com/en/rest/migrations/users?apiVersion=2022-11-28#start-a-user-migration
+     *
+     * @param array $params
+     *
+     * @throws MissingArgumentException
+     *
+     * @return array|string
+     */
+    public function start(array $params)
+    {
+        if (!isset($params['repositories'])) {
+            throw new MissingArgumentException(['repositories']);
+        }
```

Comment:
Please remove this param check, we don't do this anymore in the api code but let github do the validation and throw an error response